doc/source/myrocks/zenfs.rst

```diff
@@ -201,7 +215,19 @@ Configuration
 
    .. code-block:: bash
 	
-	  $ sudo ps-admin --enable-rocksdb -u root -p <password>
+	  $ sudo ps-admin --enable-rocksdb -u root -p
+
+   And enter the MyRocks password.
+
+   Alternatively RocksDB can be enabled by adding the following lines to the mysqld.cnf file:
```

Comment:
Alternatively (add a comma)
Avoid using the passive tense (RocksDB can be enabled). Maybe 
Alternatively, enable RocksDB by adding ...